data/pid_data.py

```diff
@@ -126,7 +126,35 @@
        'value': 0x8005},
       ]
   },
-
+  # Ayrton
+  {'id': 16761,
+   'name': 'Ayrton',
+   'pids': [ { 'get_request': { 'items': []},
+               'get_response': { 'items': [{ 'name': 'dimmer_mode', 'type': 'bool'}]},
```

Comment:
Using a bool is odd. What's the difference between DIMMER_MODE on vs off? Should this be labels values instead?